doc/source/release-notes/Percona-Server-5.6.30-76.3.rst

```diff
@@ -0,0 +1,52 @@
+.. rn:: 5.6.30-76.3
+
+==============================
+ |Percona Server| 5.6.30-76.3 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.30-76.3 on May 25th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.30-76.3/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.30 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-30.html>`_, including all the bug fixes in it, |Percona Server| 5.6.30-76.3 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.30-76.3 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.30-76.3>`_.
+
+Bugs Fixed
+==========
+ 
+ When :ref:`tokudb_read_free_replication` was enabled for |TokuDB| and there was no explicit primary key for the replicated |TokuDB| table there could be duplicated records in the table on update operation. Bug fixed :bug:`1536663` (:tokubug:`950`).
+
+ Attempting to execute a non-existing prepared statement with :ref:`response_time_distribution` plugin enabled could lead to a server crash. Bug fixed :bug:`1538019`.
+
+ |TokuDB| was using using different memory allocators this was causing ``safemalloc`` warnings in debug builds and crashes because memory accounting didn't add up. Bug fixed :bug:`1546538` (:tokubug:`962`).
+
+ Buffer pool may fail to remove dirty pages for a particular tablespace from the flush list. Bug fixed :bug:`1552673`.
+
+ Fixed heap allocator/deallocator mismatch in :ref:`scalability_metrics_plugin`. Bug fixed :bug:`1581051`.
+
+ |Percona Server| is now built with system ``zlib`` library instead of the older bundled one. Bug fixed :bug:`1108016`.
+
+ Fixed memory overhead per page in the |InnoDB| buffer pool. The fix was ported from Facebook patch `#91e979e <https://github.com/facebook/mysql-5.6/commit/91e979e8436b83400e918fa0f251036e50d0cb5f>`_. Bug fixed :bug:`1536693` (upstream :mysqlbug:`72466`).
```

Comment:
s/Fixed/Reduced
The fix was not ported exactly, but merely based on the Facebook version